index.md

```diff
@@ -57,36 +57,40 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
 * [Game of Life Kata - Ruby][]
 * [FizzBuzz Kata - Ruby][]
 * [Bowling Kata - Ruby][]
-* Tic-tac-toe Kata - Ruby
+* [Tic-tac-toe Kata - Ruby](https://gist.github.com/dlresende/dc57f506e1ab1de7e7df)
 
 **Training:**
 
-* Introduction to frontend project
-* [CodeSchool: Front-end Foundations][]: Level 2, Level 3, Level 5
+* Frontend Project: Introduction
+* [CodeSchool: Front-end Foundations][]
 * [CodeSchool: Front-end Formations][]
 * [CodeSchool: CSS Cross-Country][]
-* Frontend project style prototype review
-* Frontend project HTML review
-* Frontend project CSS review
+* Frontend Project - Review: Style Prototype
+* Frontend Project - Review: HTML
+* Frontend Project - Review: CSS
 * [CodeSchool: Assembling Sass][]
 * [CodeSchool: Assembling Sass Part 2][]
-* Introduction to Compass
-* Frontend project Sass review (CSS to be rewritten)
+* Frontend Project - Review: Sass
+  - Rewrite your CSS :art:
 * [CodeSchool: Journey Into Mobile][]
-* Frontend project RWD practices review (CSS to be rewritten)
+* Frontend Project - Review: Responsive Web Design (RWD)
+  - Rewrite your CSS :art:
 * [CodeAcademy: Introduction to JavaScript][]
 * Introduction to Jasmine
 * [FizzBuzz Kata - Javascript][] *with unit tests*
-* [CodeSchool: CoffeeScript][]
-* [CodeSchool: jQuery Air: First Flight][]
-* [CodeSchool: jQuery Air: Captain's Log][]
-* Frontend project JavaScript review (CSS to be rewritten)
-* Frontend project final review and launch!
+* [Javascript Road Trip Part I](https://www.codeschool.com/courses/javascript-road-trip-part-1)
+* [Javascript Road Trip Part II](https://www.codeschool.com/courses/javascript-road-trip-part-2)
+* [Optional: Javascript Road Trip Part III](https://www.codeschool.com/courses/javascript-road-trip-part-3)
+* Frontend Project - Review: JavaScript
```

Comment:
I am not sure the Frontend Project actually has JS in its current form, @zastrow ?